app.js

```diff
@@ -24,17 +25,21 @@ app.get('/', function get(req, res) {
   }));
 });
 
-app.get('/responses', function get(req, res) {
-return smartSurveyAPICall.getData(
-    config.apiToken,
-    config.apiTokenSecret,
-    config.surveyID)
-  .then(results => res.json(results))
-  // This is a placeholder for error logging. In the future, the aim
-  // is to print a friendly message to the user with the stacktrace
-  // eslint-disable-next-line no-console
-  .catch(error => console.log(error));
-});
+// Create a function that passes in a var rather than having var in the function
+// itself.  Otherwise you have a dependency of var especially if it's a config.
+// Therefore, you'll have to stub it
+function createGet(surveyID) {
+  return function get(req, res) {
+    return smartSurveyAPI.getData(surveyID)
+      .then(results => res.json(results))
```

Comment:
Ah, yes. So either keep the lambda or bind it.

I'd say that ugly things like this are an example of OO being a bad choice in JS. You wouldn't have this problem with a closure, for example.